_sources/examples.rst.txt

```diff
@@ -0,0 +1,6 @@
+Examples
+=========
+
```

Comment:
The examples page should be more descriptive, if not a full tutorial.

Reply:
@anshchaube 
Commit df2c7fe
Please check out changes here: https://andrewryh.github.io/saltproc/examples.html
Don't wanna add a detailed description of input syntax yet, because it is constantly changing.

Reply:
Looks good. Minor correction - "...and reprocessing system **components'** ..."

Reply:
Thanks for catching this! Fixed 
82c7cae